DESCRIPTION

```diff
@@ -1,7 +1,46 @@
+Type: Project
+Description: Residential valuation model of the Cook County Assessor's Office
+Depends:
+    arrow,
+    assessr,
+    aws.s3,
+    aws.ec2metadata,
+    butcher,
+    ccao,
+    conflicted,
+    dplyr,
+    furrr,
+    git2r,
+    glue,
+    hardhat,
+    here,
+    knitr,
+    parsnip,
+    purrr,
+    lightgbm,
+    lightsnip,
+    lubridate,
+    paws.analytics,
+    paws.application.integration,
+    recipes,
+    rlang,
+    rsample,
+    stringr,
+    tictoc,
+    tidyr,
+    tune,
+    workflows,
+    yaml,
+    yardstick
+Remotes:
+    ccao-data/assessr,
+    ccao-data/ccao,
+    ccao-data/lightsnip
```

Comment:
Since I consolidated all the dependencies to `R/setup.R` and we explicitly track the dev dependencies via profiles, I figured it can't hurt to just make all our dependency tracking explicit.